src/native/external/zstd-version.txt

```diff
@@ -0,0 +1,8 @@
+v1.5.7
+https://github.com/facebook/zstd/releases/tag/v1.5.7
+
+Only the 'lib' and 'build' directories from the original source were included to minimize unnecessary code.
```

Comment:
Care to add a checklist for taking updates?  It should mention which commands to run, which additional files to update, frequency, source location, validation critieria, etc.